src/lib/hooks/useScrollPositionHook.js

```diff
@@ -0,0 +1,31 @@
+import { useLayoutEffect } from 'react';
+import { getElementsPositionDifference } from '../services/elementPositionService';
+
+export const useScrollPosition = (effect, dependencies, contentEl, viewportEl, wait) => {
+  let throttleTimeout = null;
+
+  const callBack = () => {
+    effect(getElementsPositionDifference(contentEl, viewportEl));
+    throttleTimeout = null;
+  };
+
+  useLayoutEffect(() => {
+    const handleScroll = () => {
+      if (wait) {
+        if (throttleTimeout === null) {
+          // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I don't think that //`eslint-disable-next-line react-hooks/exhaustive-deps `goes here

Reply:
![image](https://user-images.githubusercontent.com/5614085/85133722-886ed180-b23b-11ea-84c9-a5a04b47e9f4.png)

Reply:
This is still not resolved. It is necessary to use `useRef` as I wrote above. It is same for `useLoadResizeHook.js`

Reply:
I just checked the ESlint rule, not the error itself. Fixed.